daemon/algod/api/spec/v1/model.go

```diff
@@ -675,6 +675,11 @@ type AssetTransferTransactionType struct {
 	//
 	// required: false
 	CloseTo string `json:"closeto"`
+
+	// CloseToAmount is amount of the remaining funds that were transferred to the close to address (if closing).
+	//
+	// required: false
+	CloseToAmount uint64 `json:"close-to-amount"`
```

Comment:
nit: maybe `AssetClosingAmount` ? we already have a field called `AssetID`, so the prefix "Asset" isn't foreign.

Reply:
it's `v1` I just matched it to the rest. That being said, I did change `close-amount-to` to `closeamountto` to match the pattern there.